website/sidebars.json

```diff
@@ -63,7 +63,8 @@
       "deploy/google_auth",
       "deploy/database_backup",
       "deploy/migration_guide",
-      "deploy/import_export"
+      "deploy/import_export",
+      "deploy/technical_debt"
```

Comment:
we are not very consistent on the "_" vs "-" today, but it will become part of the url. Personally i prefer `http://some/simple-read` over `http://some/simple_read`

Reply:
Agree, it looks a lot nicer in the url. Let's put it up as task.